### DISCUSSION TITLE
Is there any convenient script to generate STUR file from 'cif' or 'xyz' or other formate?

### DISCUSSION THREAD

**Discussion by StealFox97** [deepmodeling/abacus-develop#2871] [Q&A] [answered]
Dear developer,
 I'm new to this software. The ATOMIC_POSITION section is different from other software. Is there any convenient script to generate STUR file from 'cif' or 'xyz' or other formate?

**Comment by LiangWenshuo1118** [comment, marked as answer]
You can visit the link: https://nb.bohrium.dp.tech/detail/9814968648 to get the solution.

Note: 
You need to first register a Bohrium account at https://bohrium.dp.tech/login?pageType=register in order to access it smoothly.

**Comment by QuantumMisaka** [comment]
You can use ASE-ABACUS interface like the last comment or use ATOMKIT

**Reply by StealFox97** [reply]
Thank you for your advice! It took a bit of time to install ase-abacus, but it works well!
